### DISCUSSION TITLE
Is it possible to see profile pictures from routed emails?

### DISCUSSION THREAD

**Discussion by Vnzq4Qje3zzU** [simple-login/app#859] [Q&A]
There is no standard to show profile pictures and most of those that shows something will rely on the sender address, which SL changes. Is there anyway to bypass it? Maybe adding a mode where email sender isn't modified,  like Cloudflare (which will keep the "From:" field), and let the user deal with the replies by another more weird method (ideal for those that hardly ever reply to something).

**Comment by nguyenkims** [comment]
We did indeed plan to use Gravatar but finally abandon this as we don't want to make a request to an external service to avoid leaking user info.
